jrnl/cli.py

```diff
@@ -53,26 +54,183 @@ def parse_args(args=None):
     return parser.parse_args(args)
 
 
-def guess_mode(args, config):
-    """Guesses the mode (compose, read or export) from the given arguments"""
-    compose = True
-    export = False
-    import_ = False
-    if args.import_ is not False:
-        compose = False
-        export = False
-        import_ = True
-    elif args.decrypt is not False or args.encrypt is not False or args.export is not False or any((args.short, args.tags, args.edit)):
-        compose = False
-        export = True
-    elif any((args.start_date, args.end_date, args.on_date, args.limit, args.strict, args.starred)):
+class CommandEnum:
```

Comment:
Thanks for the comment.  I made CommandEnum a new-style class to keep things simple.  You will find the new commit in this pull request.